iroh-net/src/endpoint.rs

```diff
@@ -1247,14 +1245,10 @@ fn proxy_url_from_env() -> Option<Url> {
 /// Otherwise, it will return `RelayMode::Default`.
 pub fn default_relay_mode() -> RelayMode {
     // Use staging in testing
-    #[cfg(not(any(test, feature = "test-utils")))]
     let force_staging_relays = match std::env::var(ENV_FORCE_STAGING_RELAYS) {
         Ok(value) => value == "1",
```

Comment:
should be any non-nil value.  Maybe we can add a `pub(crate)` function to read this variable and also put it in `src/relay.rs`.  Otherwise this behaviour gets to easily different between different call sites.